src/pages/Event/Event2023FallMissions.tsx

```diff
@@ -19,38 +35,77 @@ const MissionContainer = () => {
     border: `2px solid #EEEEEE`,
     borderRadius: "10px",
     overflow: "hidden",
+    marginRight: "12px",
+    backgroundColor: theme.white,
   };
-  const styleDescription = {
+  const styleContentBox = {
     width: 0,
     flexGrow: 1,
+  };
+  const styleTitle = {
+    ...theme.font16_bold,
+    color: isDone ? theme.gray_text : theme.black,
+    marginBottom: "4px",
+  };
+  const styleDescription = {
     ...theme.font12,
-    color: theme.gray_text,
+    color: isDone ? theme.gray_text : theme.black,
+  };
+  const styleReward = {
+    display: "flex",
+    justifyContent: "space-between",
+    marginTop: "12px",
   };
 
   return (
-    <WhiteContainer css={{ padding: "12px 12px 12px 20px" }}>
+    <WhiteContainer
+      css={{
+        padding: "12px 12px 12px 20px",
+        backgroundColor: isDone ? theme.gray_background : theme.white,
+      }}
+    >
       <div
         css={{
           position: "absolute",
           top: 0,
           bottom: 0,
           left: 0,
           width: "8px",
-          background: theme.purple,
+          background: isDone ? theme.purple_disabled : theme.purple,
         }}
       />
       <div css={styleBody}>
-        <div css={styleImageWrap}></div>
-        <div css={styleDescription}>
-          2명 이상 탑승하고 정산 완료하기 여기는 미션 설명을 위한 공간입니다
-          최대 세 줄까지 들어가도록 해볼게요
+        <div css={styleImageWrap}>
+          <img src={quest?.imageUrl} alt={quest?.name} />
+        </div>
+        <div css={styleContentBox}>
+          <div css={styleTitle}>{quest?.name}</div>
+          <div css={styleDescription}>{quest?.description}</div>
         </div>
       </div>
+      {!isDone ? (
+        <div css={styleReward}>
+          <div css={{ ...theme.font12 }}>
+            달성 {questCompletedCnt}번 / 최대 {quest?.maxCount}번
+          </div>
+          <div
+            css={{
+              display: "flex",
+              width: "57px",
+              justifyContent: "space-between",
+            }}
+          >
+            <CreditIcon css={{ width: "27px", height: "16px" }} />
+            <div css={{ ...theme.font12 }}>{quest?.reward?.credit}</div>
+          </div>
+        </div>
+      ) : null}
```

Comment:
이런식으로도 코드 작성 가능해요!! 뒤에 `null` 을 붙이지 않으셔도 됩니다!

```suggestion
      {!isDone && (
        <div css={styleReward}>
          <div css={{ ...theme.font12 }}>
            달성 {questCompletedCnt}번 / 최대 {quest?.maxCount}번
          </div>
          <div
            css={{
              display: "flex",
              width: "57px",
              justifyContent: "space-between",
            }}
          >
            <CreditIcon css={{ width: "27px", height: "16px" }} />
            <div css={{ ...theme.font12 }}>{quest?.reward?.credit}</div>
          </div>
        </div>
      )}
```